CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/WaitForSmokeTestToFinishModule.cs

```diff
@@ -111,6 +120,12 @@ private void ValidateInputArguments()
 			{
 				throw new ArgumentNullException(nameof(WorkspaceName), $"{nameof(WorkspaceName)} cannot be NULL or Empty.");
 			}
+
+			int timeoutValue;
+			if (string.IsNullOrWhiteSpace(TimeoutValueInMinutes) || !int.TryParse(TimeoutValueInMinutes, out timeoutValue))
+			{
+				throw new ArgumentNullException(nameof(TimeoutValueInMinutes), $"{nameof(TimeoutValueInMinutes)} cannot be NULL, Empty, or an invalid Guid.");
```

Comment:
we need to update the error message. it mentioned a GUID.

Reply:
Also it is a good idea to split the if condition into two. One with ArgumentNullException for the first part and ArgumentException for the second part to be more clear.